CMakeLists.txt

```diff
@@ -690,7 +695,16 @@ macro(srt_add_application name sources)
 	install(TARGETS ${name} RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
 endmacro()
 
-if ( ENABLE_CXX11 )
+## FIXME: transmitmedia.cpp does not build on OpenBSD as some of the Multicast
+##    socket operations used there are non-portable.
+##    See https://man.openbsd.org/multicast
+if (BSD
+   AND ${CMAKE_SYSTEM_NAME} MATCHES "^[oO][pP][eE][nN][bB][sS][dD]$")
+   set(DISABLE_SUPPORT_APPLICATIONS ON)
+endif()
+
+if ( ENABLE_CXX11
+   AND NOT DISABLE_SUPPORT_APPLICATIONS )
```

Comment:
For me this looks like two separate variables for the same purpose. My proposal is: make the `ENABLE_CXX11` option control the other, and here only the other option shall be in the condition.

Reply:
@jlsantiago0 
Modify line 248 (add `set(ENABLE_APPS 0)`):
```
if (NOT ENABLE_CXX11)
	set(ENABLE_APPS 0)
	message(WARNING "Parts that require C++11 support will be disabled (srt-live-transmit)")
endif()
```

Reply:
It feels to me that using a variable ENABLE_CXX11 to determine select building of the sample/support applications is the problem. Not the ENABLE_APPS or ENABLE/DISABLE_SUPPORT/SAMPLE_APPLICATIONS/APPS. In otherwords the decision to ENABLE_APPS depends on ENABLE_CXX11, and then the guard to actually do the building of the applications should be using ENABLE_APPS or whatever.

Reply:
I just checked in a change to use the ENABLE_APPS as the guard to build the applications that is set to OFF when ENABLE_CXX11 is OFF.

Reply:
I agree that ENABLE_CXX11 is now redundant since enabling APPS or TESTS implicitly means CXX11. We should probably have a statement for backward compatibility where disabling CXX automatically means disabling APPS and TESTS, but, otherwise, ENABLE_CXX11 should not be used anywhere in the make logic.

Reply:
OK. I committed a change to resolve this issue.